### PR TITLE
Add tokens to uast parser

### PR DESCRIPTION
**Pull request by alcortesm**
merge https://github.com/bblfsh/bash-driver/pull/11 first.

This allows the driver to collect the original text added in https://github.com/bblfsh/bash-driver/pull/11 to the native parser.

### CODE REVIEW

File: native/src/main/java/bblfsh/bash/ASTNodeSerializer.java [hunk 1]
```diff
@@ -35,6 +35,9 @@ public void serialize(ASTNode root, JsonGenerator jG, SerializerProvider provide
         final int length = root.getTextLength();
         jG.writeNumberField("textLength", length);
 
+        final String text = root.getText();
+        jG.writeStringField("text", text);
```
Comment (bzz) [inline native/src/main/java/bblfsh/bash/ASTNodeSerializer.java]: This and below seems very similar to #11 - is that expected?

Comment (alcortesm) [inline native/src/main/java/bblfsh/bash/ASTNodeSerializer.java]: Yes, this PR and #11 has a commit in common, which is why you see the changes twice.

I always get confused whenever this happens too and that's why I started this PR's text explaining that it is related with #11.

I would like to stop doing this kind of PR mixing, but I don't know how to do it without delaying PRs until their dependencies had been merged.  The rest of the team seems to be ok with this kind of workflow anyway.

Comment (bzz) [inline native/src/main/java/bblfsh/bash/ASTNodeSerializer.java]: Got it, thanks! I guess the only way is to merge PRs fast..

Comment (alcortesm) [inline native/src/main/java/bblfsh/bash/ASTNodeSerializer.java]: Yes, sadly that is the same conclusion I reached.

**Comment by bzz** [comment]
Looks great to me.

File: driver/normalizer/fixtures/var.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (smola) [inline driver/normalizer/fixtures/var.json]: This kind of fixture should be in the `test` directory, so that it can be used in integration tests.

Comment (alcortesm) [inline driver/normalizer/fixtures/var.json]: Understood, will do it in a future PR.

**Pull request merged**